classes.py

```diff
@@ -336,13 +336,177 @@ def display(self):
                 legendary_action.display()
 
 
+def redundant(word): #Skip redundent expressions like repeats on rarity, properties, etc.
```

Comment:
Move redundant() to utils if you plan to use it later or remove it if you don't have a use for it.